README.md

```diff
@@ -16,7 +16,12 @@ Better [Markdown](http://en.wikipedia.org/wiki/Markdown) parser for PHP.
 
 ### Installation
 
-Include `Parsedown.php` or install [the composer package](https://packagist.org/packages/erusev/parsedown).
+Download and include [Parsedown.php] or install the [composer package] by running the following command:
+
+    composer require erusev/parsedown
+
+[Parsedown.php]: https://raw.githubusercontent.com/erusev/parsedown/master/Parsedown.php "The Parsedown class file"
```

Comment:
The only thing I'm hesitating on here is encouraging downloading straight from `master`. Probably safer for the manual instructions is saying something like:
```markdown
1. Download "Source code" from the [latest release](https://github.com/erusev/parsedown/releases/latest)
2. Include `Parsedown.php`
```

Reply:
fine with me, feel free to cherry-pick the commit and adjust it.

Reply:
Mind checking the box in the sidebar that let's me push changes to this PR? :)